### PR TITLE
add builtin pipelines for python

### PR DESCRIPTION
**Pull request by imjasonh**
Also use them in `melange convert python`, resolving a TODO.

### CODE REVIEW

File: pkg/build/pipelines/python/build-wheel.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Build a Python wheel
+
+needs:
+  packages:
+    - busybox
+
+pipeline:
+  - runs: |
+      if ! [ -x "$(command -v python)" ]; then
```
Comment (kaniini) [inline pkg/build/pipelines/python/build-wheel.yaml:9]:
```suggestion
      if ! [ -x "$(command -v python3)" ]; then
```

File: pkg/build/pipelines/python/build.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: Build a Python wheel
```
Comment (vaikas) [inline pkg/build/pipelines/python/build.yaml:1]: What's the difference between this and build-wheel above?

File: pkg/build/pipelines/python/build-wheel.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: Build a Python wheel
```
Comment (vaikas) [inline pkg/build/pipelines/python/build-wheel.yaml:1]: Could we add some comments here on when one would use which variant? Even if it's heuristics?

File: pkg/build/pipelines/python/build.yaml (hunk 2 above)
Comment (imjasonh) [reply pkg/build/pipelines/python/build.yaml:1]: build-wheel uses python3 specifically to build a .whl file. This `build` seems to basically just copy .py files into the right place, which is as close as Python gets to "building" things. 🤷

File: pkg/build/pipelines/python/build-wheel.yaml (hunk 3 above)
Comment (imjasonh) [reply pkg/build/pipelines/python/build-wheel.yaml:1]: I would, but I don't actually know myself. There seem to be 80 ways of packaging stuff in Python, and this is just a few of them. Each project seems to do it a slightly different and fun way.

**Review by vaikas — APPROVED** [review]

**Pull request merged**